extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/search/YoutubeMusicSearchExtractorTest.java

```diff
@@ -0,0 +1,63 @@
+package org.schabi.newpipe.extractor.services.youtube.search;
+
+import org.junit.BeforeClass;
+import org.schabi.newpipe.DownloaderTestImpl;
+import org.schabi.newpipe.extractor.InfoItem;
+import org.schabi.newpipe.extractor.NewPipe;
+import org.schabi.newpipe.extractor.StreamingService;
+import org.schabi.newpipe.extractor.search.SearchExtractor;
+import org.schabi.newpipe.extractor.services.DefaultSearchExtractorTest;
+import org.schabi.newpipe.extractor.services.youtube.linkHandler.YoutubeSearchQueryHandlerFactory;
+
+import java.net.URLEncoder;
+
+import javax.annotation.Nullable;
+
+import static java.util.Collections.singletonList;
+import static org.schabi.newpipe.extractor.ServiceList.YouTube;
+
+public class YoutubeMusicSearchExtractorTest {
```

Comment:
This is not testing all available filters for YouTube Music, it should test all of them.